### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by 0sugo**
In this PR I did the following:

- [x] Setup a new React application using [Create React App]

- [x] Created a directory for the reusable components: /components.

- [x] Installed React Router V6

- [x] Setup your router inside of <App>

- [x] Added 2 routes(Books,Categories) in the application, each  rendering individual  page component:

- [x] Created components (Categories, Home, and Nav)inside of the /components directory responsible for rendering:

  - Individual book state, as well as a button to delete the book
  - A list to render individual books
  - A  form to create new books
  - A navigation to link to the 2 routes you created

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const Home = () => (
```
Comment (divinecharlotte) [inline src/components/Home.js]: - [x] Kindly consider adding A <Books> component, which contains the following components:


 `<BookList>`

- Renders a list of books using the <Book> component



`<Book>`

- Takes the book data as a prop and renders the title and the author

- Has a button for removing books

💀 the button wouldn't remove the books yet, this is just a placeholder for now

doing so will help you to meet the below requirements

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          <option value="Biography">Biography</option>
+          <option value="History">History</option>
+          <option value="Horror">Horror</option>
+          <option value="Kids">Kids</option>
+          <option value="Learning">Learning</option>
+          <option value="Sci-Fi">Sci-Fi</option>
+        </select>
+        <button type="submit" className="book-adder">ADD BOOK</button>
+      </form>
+    </div>
```
Comment (divinecharlotte) [inline src/components/Home.js]: - [x] kindly consider separating this into its own component to meet the below requirement and have well organised codes
Screen Shot 2023-05-06 at 02 41 15

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @0sugo     , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- Clean code 👍
- linters are passing 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @0sugo , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**